database/factories/ModelFactory.php

```diff
@@ -46,6 +46,7 @@
 $factory->define(Signup::class, function (Generator $faker) {
     $faker->addProvider(new FakerNorthstarId($faker));
     $faker->addProvider(new FakerCampaignId($faker));
+    $details_options = [null, null, 'fun-affiliate-stuff', 'i-say-the-tails'];
```

Comment:
Might want to add a note about the extra `null` values. I tried to figure out if there was a nice way of making sure something is picked more than others, but this seems fine to me.

Reply:
I could also remove it, I don't think the proportions will matter so much for this? And it does look pretty funny.

Reply:
yea it's probably not that important!